src/Nowruz/modules/Auth/contexts/onboarding/sign-up-user-onboarding.context.tsx

```diff
@@ -16,29 +16,23 @@ const initialState = {
   username: '',
   avatar: '',
   address: '',
-};
-const orgInitialState = {
   orgName: '',
   orgType: { value: 'STARTUP', label: 'Impact Startup' },
-  social_causes: [],
-  bio: '',
   image: '',
-  city: '',
-  country: '',
-  cityLabel: '',
   email: '',
   website: '',
   size: null,
   shortname: '',
   industry: '',
 };
+
 let type = localStorage.getItem('registerFor');
 
 const reducer = (state, action) => {
   switch (action.type) {
-    case 'UPDATE_USER':
+    case 'UPDATE_USER': {
       const filteredPayload = Object.keys(action.payload).reduce((filtered, key) => {
-        if (state.hasOwnProperty(key)) {
+        if (Object.prototype.hasOwnProperty.call(state, key)) {
```

Comment:
why use `call`?

Reply:
esLint error

Reply:
`state.hasOwnProperty` causes ESlinit errors in commit. I applied `getOwnPropertyDescriptor` to avoid both `call` and ESlint error